test/components/cert-manager/cert-manager_test.go

```diff
@@ -47,7 +46,7 @@ func TestCertManagerDeployments(t *testing.T) {
 	for _, test := range testCases {
 		t.Run(fmt.Sprintf("cert-manager deployment:%s", test.deployment), func(t *testing.T) {
 			t.Parallel()
-			testutil.WaitForDeployment(t, client, namespace, test.deployment, time.Second*5, time.Minute*5)
+			testutil.WaitForDeployment(t, client, namespace, test.deployment, testutil.RetryInterval, testutil.Timeout) //nolint:goscope
```

Comment:
It seems we should add `test := test` after for-range, rather than `//nolint:goscope`.